clusterloader2/testing/density/config.yaml

```diff
@@ -86,30 +69,22 @@ steps:
       templateFillMap:
         Replicas: 1
         PodKind: latencyPod
-- measurements:
+- name: Wait for latency pods
```

Comment:
And this should be fairly fast - it should be pretty much <5s according to our SLO :-)

Reply:
So in other words - I don't think these two provide any value.